staging/src/k8s.io/metrics/pkg/apis/metrics/v1alpha1/types.go

```diff
@@ -29,26 +29,26 @@ import (
 // resource usage metrics of a node.
 type NodeMetrics struct {
 	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
 	// The following fields define time interval from which metrics were
 	// collected from the interval [Timestamp-Window, Timestamp].
-	Timestamp metav1.Time     `json:"timestamp"`
-	Window    metav1.Duration `json:"window"`
+	Timestamp metav1.Time     `json:"timestamp" protobuf:"bytes,2,opt,name=timestamp"`
+	Window    metav1.Duration `json:"window" protobuf:"bytes,3,opt,name=window"`
 
 	// The memory usage is the memory working set.
-	Usage v1.ResourceList `json:"usage"`
+	Usage v1.ResourceList `json:"usage" protobuf:"bytes,4,rep,name=usage,casttype=k8s.io/client-go/pkg/api/v1.ResourceList,castkey=k8s.io/client-go/pkg/api/v1.ResourceName,castvalue=k8s.io/apimachinery/pkg/api/resource.Quantity"`
```

Comment:
ResourceList will be moved k8s.io/api in #44784, shall i just change the package name in casttype and castkey in #44784?